doc/bt/common.bt

```diff
@@ -112,11 +113,30 @@ typedef struct
     int i7;
     short hair;
     short pose;
-
-    int i8;
+    int teamId;
     int i9;
 }
-COMMANDER;
+APPEARANCE_PC;
```

Comment:
The `APPEARANCE_PC` structure is also a component of the `APPEARANCE_BARRACK_PC` and `BARRACK_PC` structures.